week-2/14-align-self/flexbox.css

```diff
@@ -0,0 +1,13 @@
+.countries {
+	display: flex;
+}
+/* Add your own CSS code below */
+
+.countries--first {
+	flex-direction: column;
+	align-items: flex-start;
+}
+
+.countries--first #uganda {
+	align-self: center;
+} 
```

Comment:
Another `.DS_Store` file (below). Is there a `.gitignore` rule we should add?

Reply:
yes, I'll add one 👍